Source/SIMPLib/ITK/itkImageReaderHelper.cpp

```diff
@@ -253,12 +244,14 @@ ::readImageOutputInformation(const DataArrayPath& dataArrayPath, typename itk::I
   // Initialize torigin/tspacing/tDims since arrays are always of size 3 and ITK image may have a different size.
   FloatVec3Type torigin = {0.0f, 0.0f, 0.0f};
   FloatVec3Type tspacing = {1.0f, 1.0f, 1.0f};
-  SizeVec3Type tDims = {1, 1, 1};
+  SizeVec3Type tDims = {0, 0, 0};
+  QVector<size_t> qTdims;
   for(size_t i = 0; i < dimensions; i++)
   {
     torigin[i] = origin[i];
     tspacing[i] = spacing[i];
     tDims[i] = size[i];
+    qTdims.push_back(tDims[i]);
```

Comment:
`push_back` isn't the fastest operation when you're increasing the size of the underlying array (should average out to O(N log(N))), so if you're importing a lot of data, you might want to use `QVector<size_t> qTdims(dimensions);` with `qTdims[i] = tDims[i];`

Reply:
O(N log(N)) is a best case scenario, but it depends on implementation.  Qt collections are known to be slower than their STL counterparts, so it would be best to make it as constant-time as possible.

Reply:
In the context of the code the largest the array will be is 3, maybe 4.

Reply:
Does itkImageReaderHelper get used for stacks of images?  My guess is that the initial underlying array is size 0, so you're probably having two resizes for each image.

Reply:
It is only used in ITKImageReader

Reply:
Any other comments on this PR?  I want to get it merged in if we are ok with it.  Right now, importing 2D images (including montage tiles) is broken, and this fixes it.